README.md

```diff
@@ -291,6 +292,18 @@ Metrics are used to monitor the health of your fleet across components. Continue
 | node:node_net_utilisation:sum_irate                                      |         :x:        |         :x:        |         :x:        |         :x:        |
 | 'node_namespace_pod:kube_pod_info:'                                      |         :x:        |         :x:        |         :x:        |         :x:        |
 
+### Server foundation metrics
+
+| Metric Name                                                                  | ACM 2.5            | ACM 2.6            | ACM 2.7            | ACM 2.8            |
+|------------------------------------------------------------------------------|--------------------|--------------------|--------------------|--------------------|
+| acm_managed_cluster_labels                                                   |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_managed_cluster_info                                                     |         :x:        |         :x:        |         :x:        | :white_check_mark: |
+| acm_managed_cluster_count                                                    |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_managed_cluster_addon_status_condition                                   |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_managed_cluster_status_condition                                         |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_manifestwork_status_condition                                            |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_manifestwork_count                                                       |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+
```

Comment:
I struggle bit with the metrics like above that are not exposed. It is provided in 2.7 tho, so that is correct. 
So should we add another column to say that these are not exposed? Or that will be too much? Thoughts - @zhiweiyin318 @cadawson @elgnay

Reply:
Thanks @bjoydeep. We could add "(not exposed)" beside the ones that are not exposed, though I don't know how much of an effort that would involve this close to GA. The reason being, does that open the door to other metrics in the list that are unexposed and not currently noted? The assumption then becomes that every metric that is not exposed is noted, and it might not be. We could also remove the table from this page for now, and try to add it later with exposed or not exposed noted somehow. Or we can leave it as it is for now. 

Let's see what @zhiweiyin318 and @elgnay think.

Reply:
Thanks @zhiweiyin318. Are you ok with that, @bjoydeep?

Reply:
sounds reasonable.